openpype/hosts/houdini/plugins/load/load_fbx.py

```diff
@@ -16,7 +16,7 @@ class FbxLoader(load.LoaderPlugin):
 
     order = -10
 
-    families = ["staticMesh", "fbx"]
+    families = ["model", "staticMesh", "fbx"]
     representations = ["fbx"]
```

Comment:
Another approach could be doing:
```suggestion
    families = ["*"]
    representations = ["*"]
    extensions = {"fbx"}
```
E.g. like this: https://github.com/ynput/OpenPype/blob/277a0baa7bbe89b2a58fbe9568271e4d5f72e86b/openpype/hosts/fusion/plugins/load/load_usd.py#L19-L21

Anyway, that's usually the way to go if the loader is supposed to be able to just load any FBX - no matter the family.

Reply:
so cool!